storage/storage_transport.go

```diff
@@ -238,7 +266,9 @@ func (s storageTransport) ValidatePolicyConfigurationScope(scope string) error {
 	}
 	storeSpec := scope[1:closeIndex]
 	scope = scope[closeIndex+1:]
-	storeInfo := strings.SplitN(storeSpec, "@", 2)
+	optionInfo := strings.SplitN(storeSpec, ":", 2)
+	runRootInfo := strings.SplitN(optionInfo[0], "+", 2)
+	storeInfo := strings.SplitN(runRootInfo[0], "@", 2)
```

Comment:
No, it shouldn't have had to.  This is left over from an earlier version of the patch where they did.  I'll undo this part.